style.css

```diff
@@ -0,0 +1,1980 @@
+    padding: 0;
+    display: flex;
+  }
+
+  .ul-tecnologies-works-6 li {
+    background-color: rgba(255, 255, 255, 0.24);
+    margin-right: 12px;
+    padding: 10px 12px;
+    width: 55px;
+    height: 20px;
+    list-style: none;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: 'Inter'sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .works-6:hover {
+    background-image: url(./images/Img\ work-01.png);
+    border: 1px solid #1c1a19;
+    justify-content: end;
+
+    .see-Proyect-works-2 {
+      display: block;
+    }
+
+    .subtitle-works-6 {
+      opacity: 0;
+    }
+
+    .paragraf-works-6 {
+      opacity: 0;
+    }
+
+    .ul-tecnologies-works-6 {
+      opacity: 0;
+    }
+  }
+
+  /* aboutme  */
+
+  .aboutme {
+    display: flex;
+    flex-direction: column;
+    padding-left: 10%;
+    color: #fff;
+    background-color: #1c1a19;
+    width: 100%;
+    height: auto;
+    background-image: url(./images/Frame2B.png);
+    background-repeat: no-repeat;
+    background-size: auto;
+    background-position: right bottom 0;
+  }
+
+  .aboutme h2 {
+    color: #ff6b00;
+    padding-top: 130px;
+    padding-left: 20px;
+    font-family: 'Crete Round', serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+  }
+
+  .aboutme p {
+    width: 40%;
+    margin-top: 20px;
+    padding-left: 20px;
+    color: #f4f5f7;
+    font-family: 'Inter'sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 24px;
+  }
+
+  .vector-aboutme {
+    margin-left: 10%;
+    width: 80%;
+    height: 1px;
+    margin-top: 170px;
+  }
+
+  .resume-button {
+    cursor: pointer;
+    background-color: #ff6b00;
+    color: #fff;
+    margin-top: 24px;
+    margin-bottom: 90px;
+    margin-left: 20px;
+    width: 156px;
+    height: 48px;
+    font-family: 'Inter'sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+  }
+
+  .aboutme-habilities {
+    width: 80%;
+    margin-left: 10%;
+    padding-top: 124px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: flex-start;
+  }
+
+  /* abutme-contact  */
+
+  .aboutme-contact {
+    /* overflow-x: hidden; */
+    display: flex;
+    flex-direction: row;
+    align-items: baseline;
+    background-color: #fff;
+    padding-left: 5%;
+  }
+
+  .aboutme-contact2 {
+    /* overflow-x: hidden; */
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    background-color: #fff;
+    width: 60%;
+    padding-left: 5%;
+  }
+
+  .aboutme-contact p {
+    overflow-x: hidden;
+    display: flex;
+    flex-direction: column;
+    max-width: 34%;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
```

Comment:
- [ ] [OPTIONAL] 

I kindly suggest that you please kindly make sure that you align your contact form paragraph to the left as seen in the Figma design

|  |   |
| ------------- | ------------- |
| Your Design  | <img width="910" alt="image" src="https://github.com/Fernando426/Portfolio-mobile-website2/assets/52934128/997b218e-a40e-45bf-b446-a739c2ba17bf"> |
| Figma Design | <img width="597" alt="image" src="https://github.com/Fernando426/Portfolio-mobile-website2/assets/52934128/5aaadbae-7563-4ae0-bbed-b88df9b9510f"> |